### DISCUSSION TITLE
Is there a way to get back to the layout shown in the picture?

### DISCUSSION THREAD

**Discussion by hnsylzg** [LazyVim/LazyVim#5053] [Q&A]
Is there a way to go back to the layout shown in the picture? The new version has a vertical pop-up panel, which I don’t like very much.
![2024-12-14_002613](https://github.com/user-attachments/assets/18cab676-b373-4e24-af2a-de2c2a56546d)

**Comment by minusfive** [comment]
Set the `preset` option of `which-key` to `"classic"`.
